docs/installation/production-rack/aws.md

```diff
@@ -54,6 +54,8 @@ The following environment variables are required:
 | **node_capacity_type**   | **on_demand**          | Can be either "on_demand" or "spot". Spot will use AWS spot instances for the cluster nodes                    |
 | **node_disk**            | **20**                 | Node disk size in GB                                                                                           |
 | **node_type**            | **t3.small**           | Node instance type. You can also pass a comma separated list of instance types                                 |
+| **power_off_schedule**   |                        | Power off schedule is specified by the user following the Unix cron syntax format. Example: "0 0 * * 6" |
```

Comment:
nit but in the cron word, can you add a link to https://crontab.guru/? I think i'd be cool